dev/CommonStyles/ToggleSwitch_themeresources.xaml

```diff
@@ -337,13 +334,38 @@
                                             </ObjectAnimationUsingKeyFrames>
                                         </Storyboard>
                                     </VisualTransition>
+                                    <VisualTransition x:Name="OnToDraggingTransition"
+                                        From="On"
+                                        To="Dragging"
+                                        GeneratedDuration="0">
+                                        <Storyboard>
+                                            <ObjectAnimationUsingKeyFrames Storyboard.TargetName="SwitchKnobBounds" Storyboard.TargetProperty="Opacity">
+                                                <DiscreteObjectKeyFrame KeyTime="0" Value="1" />
+                                            </ObjectAnimationUsingKeyFrames>
+                                            <ObjectAnimationUsingKeyFrames Storyboard.TargetName="SwitchKnobOn" Storyboard.TargetProperty="Opacity">
+                                                <DiscreteObjectKeyFrame KeyTime="0" Value="1" />
+                                            </ObjectAnimationUsingKeyFrames>
+                                            <ObjectAnimationUsingKeyFrames Storyboard.TargetName="SwitchKnobOff" Storyboard.TargetProperty="Opacity">
+                                                <DiscreteObjectKeyFrame KeyTime="0" Value="0" />
+                                            </ObjectAnimationUsingKeyFrames>
+                                        </Storyboard>
+                                    </VisualTransition>
```

Comment:
Do we need this section? I removed it, and I did not see a difference in the visuals of the ToggleSwitch:

![toggleswitch-cleaned](https://user-images.githubusercontent.com/16122379/66498173-f31d1a00-eabd-11e9-8fad-bc0e2593bba0.gif)

Reply:
I don't think you gif is exercising this transition.  This transition occur when the toggle switch is on and we begin dragging it to off.  Your gif only drags from off to on, not on to off.

---
In reply to: [333096978](https://github.com/microsoft/microsoft-ui-xaml/pull/1415#discussion_r333096978) [](ancestors = 333096978)

Reply:
![toggleswitch-cleaned](https://user-images.githubusercontent.com/16122379/66593613-9db53b80-eb97-11e9-8e9a-fd8765be26d9.gif)

There seems to be no difference to the gifs @kaiguo has uploaded I think. 😅

Reply:
Yeah, this transition is for on to off dragging. If you comment it out the knob and knobBounds colors will be different when dragging from on to off.

But your gif seems correct. Are you sure you commented out the code 😅?

Reply:
> But your gif seems correct. Are you sure you commented out the code 😅?
I deleted the section in question and did a clean and build. The behavior is still the same 🤔

Reply:
![toggle](https://user-images.githubusercontent.com/4424330/66602897-e4873f00-eb5f-11e9-926f-766c7698953d.gif)
This is how it looks on me when I remove the code...

Reply:
Hmm weird. Then I guess it's a problem on my machine.